proto/v2/convert_proto_test.go

```diff
@@ -0,0 +1,179 @@
+// Copyright 2018 The OpenZipkin Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package zipkin_proto3_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"net"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/golang/protobuf/proto"
+
+	zipkinmodel "github.com/openzipkin/zipkin-go/model"
+	zipkin_proto3 "github.com/openzipkin/zipkin-go/proto/v2"
+)
+
+func TestParseSpans(t *testing.T) {
+	now := time.Date(2018, 10, 31, 19, 43, 35, 789, time.UTC).Round(time.Microsecond)
+	minus10hr5ms := now.Add(-(10*time.Hour + 5*time.Millisecond)).Round(time.Microsecond)
+
+	// 1. Generate some spans then serialize them with protobuf
+	payloadFromWild := &zipkin_proto3.ListOfSpans{
```

Comment:
I think we could move this var our of the test for now or to a private method, it will improve readability of the test a lot. ITOH, could we add a fail test case for this?

Reply:
As requested, I've moved that variable out of the test, PTAL.

Reply:
👍

Reply:
I meant, this test ensures that right data is converted right but does not exercise that wrong data will return errors. OTOH I believe there are many ways this can fail and testing so I am OK if we test `protoSpanIDToModelSpanID`.